recipes/tqdm/meta.yaml

```diff
@@ -0,0 +1,49 @@
+package:
```

Comment:
The standard procedure is to define the version once at the top of the file with:

```
{% set version = 4.7.1 %}
```

and then use jinja templating for where you need to use it:

```
{{ version }}
```

so, 

```
package:
  name: tqdm
  version: {{ version }}
```